hack/update-vendor.sh

```diff
@@ -233,7 +233,9 @@ for repo in $(kube::util::list_staging_repos); do
   (
     echo "=== tidying k8s.io/${repo}"
     cd "staging/src/k8s.io/${repo}"
+    sed -E -i 's/^go 1\.(.*)$/go 1.21.0/' go.mod
```

Comment:
why 1.21 specifically?

Reply:
we could have picked 1.20.0 or 1.22.0, picked at random. whichever the value, the end result would be the same i think

Reply:
This value defines the floor that we're willing to accept.
We should consider what that should be.